### PR TITLE
Unit testing 2

### PR DESCRIPTION
**Pull request by Hemanth-2002**
This branch implements unit testing for all end points.

### CODE REVIEW

File: model/startDb.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"flag"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+func StartDB() (*gorm.DB, error) {
+
+	db_user := flag.String("user", "postgres", "database user")
+	db_password := flag.String("password", "MohanNeelima@01", "database password")
```
Comment (pteja1021) [inline model/startDb.go:13]: mention such passwords in a .env file. Get them using os module

File: database/database.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	GetBookings(int) []model.Booking
+	CancelBooking(int)
+}
+
+type DBClient struct {
+	Db *gorm.DB
+}
+
+func (db DBClient) CreateUser(user model.User) {
+	db.Db.Save(&user)
```
Comment (pteja1021) [inline database/database.go]: Add return type status,error for all such functions. If db call is successful return status,nil. Else return error

File: model/startDb.go (hunk 1 above)
Comment (pteja1021) [inline model/startDb.go]: Put such delicate details in .env files and fetch by os module

File: server/show.go [hunk 3]
```diff
@@ -29,8 +28,10 @@ func (s *BmsServer) GetListOfShowsByTheatre(ctx context.Context, in *pb.Theatre)
 // function to update show details on server
 func (s *BmsServer) UpdateShowDetails(ctx context.Context, in *pb.Show) (*pb.Show, error) {
 	log.Printf("update show details called")
-	s.Db.Model(&model.Show{}).Where("id=?", in.Id).Updates(model.Show{
+	Show := model.Show{
 		MovieID: int(in.GetMovieId()),
```
Comment (pteja1021) [inline server/show.go:36]: Nice idea

**Review by pteja1021 — CHANGES_REQUESTED** [review]

File: database/database.go (hunk 2 above)
Comment (Hemanth-2002) [reply database/database.go]: created .env and stored critical data into it

File: server/show.go (hunk 3 above)
Comment (Hemanth-2002) [reply server/show.go:36]: Thanks

File: database/database.go (hunk 2 above)
Comment (Hemanth-2002) [reply database/database.go]: added return types to the functions for error handling

File: model/startDb.go (hunk 1 above)
Comment (Hemanth-2002) [reply model/startDb.go]: created .env to store critical data

**Pull request merged**